scripts/benchall.py

```
@@ -52,7 +52,7 @@ def get_target_from_name(name):
         (gzip_bench, {}),
     ]
 
-def run(bencher, kwargs, bench_target, attach=False):
+def run(bencher, kwargs, bench_target, attach=False, attach_on_error=False):
```

Ruff (0.12.2)

Missing return type annotation for public function `run`

Add return type annotation: `None`

(ANN201)